scripts/uberenv/project.json

```diff
@@ -0,0 +1,9 @@
+{
+"package_name" : "raja",
+"package_version" : "develop",
+"package_final_phase" : "hostconfig",
+"package_source_dir" : "../..",
+"spack_url": "https://github.com/spack/spack",
+"spack_commit": "bc53bb9b7cbf250c98cfe77d334ed30d6b958c21",
```

Comment:
@rhornung67 
Uberenv is configured use a specific commit in Spack for the sake of reproducibility.
I used the same commit as in other projects.

The spack reference should be updated when:
- We want to keep up with spack develop or new release (bug fix, improvements).
- Uberenv is updated and relies on a newer version (new feature) in Spack.
- RAJA package changed (for example, the local copy of RAJA package should be pushed on Spack).

For that last case in particular, the local copy of the package should only be present if it is newer than the Spack version. It is then useful to test new changes in the spack package before pushing them, just like in this PR.